python/rules_engine.py

```diff
@@ -2910,7 +2910,7 @@ def q614(self):
 		field = "Age of Applicant or Borrower"
 		edit_name = "q614"
 		fail_df = self.lar_df[self.lar_df.app_age!="NA"].copy()
-		fail_df = fail_df[~(fail_df.app_age.apply(lambda x: 17 <= int(x) <= 101))]
+		fail_df = fail_df[~(fail_df.app_age.apply(lambda x: 18 <= int(x) <= 100))]
 		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
```

Comment:
Are these the min/max on platform? When I wrote this one I used some arbitrary numbers, can you check so that our code matches what Platform uses?

Reply:
The FIG actually uses an age range of 18-100 for this edit.